src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -883,11 +894,34 @@ protected function typeMultiPolygonZ(Fluent $column)
      * Format the column definition for a PostGIS spatial type.
      *
      * @param  string  $type
+     * @param  Fluent  $column
      * @return string
      */
-    private function formatPostGisType(string $type)
+    private function formatPostGisType(string $type, Fluent $column)
+    {
+        if ($column->geography !== null) {
+            return "geography($type, ".($column->projection === null ? '4326' : $column->projection).')';
```

Comment:
I'm not sure, but maybe it will be better to use `sprintf` method here for better readability?

Reply:
I actually just rewrote it with a null coalescer, for that exact reason. Still noodling about what to do on 906 for readability though.